services/blockchain-indexer/methods/dataService/network.js

```diff
@@ -29,4 +32,27 @@ module.exports = [
 		controller: getNetworkStatistics,
 		params: {},
 	},
+	{
+		name: 'network.peers',
+		controller: getNetworkPeers,
+		params: {
+			ip: { optional: true, type: 'any' },
+			networkVersion: { optional: true, type: 'any' },
+			state: { optional: true, type: 'any' },
+			height: { optional: true, type: 'any' },
+			limit: { optional: true, type: 'any' },
+			offset: { optional: true, type: 'any' },
+			sort: { optional: true, type: 'any' },
```

Comment:
Can we set a stricter type like `string`, `number`?

Reply:
Done!